sqlmesh/utils/metaprogramming.py

```diff
@@ -304,7 +304,7 @@ def __lt__(self, other: t.Any) -> bool:
 
 
 class Executable(PydanticModel):
-    payload: t.Any
+    payload: str
```

Comment:
hm i wonder why this was an any before?

Reply:
perhaps some artifact from before. As far as I can tell we always pass string in there